lib/rules/no-invalid-role.js

```diff
@@ -116,8 +216,23 @@ module.exports = class NoInvalidRole extends Rule {
           ['presentation', 'none'].includes(roleValue) &&
           ELEMENTS_DISALLOWING_PRESENTATION_OR_NONE_ROLE.includes(node.tag)
         ) {
+          if (ELEMENTS_DISALLOWING_PRESENTATION_OR_NONE_ROLE.includes(node.tag)) {
+            this.log({
+              message: createErrorMessageDisallowedRoleForElement(node.tag),
+              line: node.loc && node.loc.start.line,
+              column: node.loc && node.loc.start.column,
+              source: this.sourceForNode(node),
+            });
+          }
+        }
+
+        if (!this.config.catchInvalidRoles) {
```

Comment:
We want this rule to be prepared for future expansions, so instead of this early return if statement here, we should combine this with the actual relevant if statement below:

```
if (this.config.catchInvalidRoles && !VALID_ROLES.includes(roleValue.toLowerCase())) {
```